Activities/DocAcquire/DocAcquire/Services/VerificationService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using DocAcquire.DataContracts;
+using DocAcquire.Helpers;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DocAcquire
+{
+    public class VerificationService : IVerificationService
+    {
+        public async Task<List<DocumentExtractResponse>> GetVerifiedDataAsync(int[] documentIds, string token, string baseUrl)
+        {
+            var apiUrl = "api/External/Documents/Verified";
+
+            var jsonString = JsonConvert.SerializeObject(documentIds);
+            var content = new StringContent(jsonString, Encoding.UTF8, "application/json");
+
+            var httpClient = CustomHttpClient.GetInstance(baseUrl);
+            httpClient.DefaultRequestHeaders.Clear();
```

Comment:
if we clear these on every call it makes sense to move that code inside the GetInstance method and not have it repeat every time